.github/CODEOWNERS

```diff
@@ -183,3 +183,7 @@ Doc/howto/clinic.rst          @erlend-aasland
 
 # WebAssembly
 /Tools/wasm/                  @brettcannon
+
+# SBOM
+/Misc/sbom.spdx.json          @sethmlarson
+/Tools/build/generate_sbom.py @sethmlarson
```

Comment:
> This CODEOWNERS file contains errors

Only those with write access can be listed in CODEOWNERS.

Reply:
Yeah I was just about to comment that I'm not a core developer myself so can't currently be added to CODEOWNERS.

Reply:
Alternatively, we can make you a triager: https://github.com/python/core-workflow/issues/515 ✅ 

> This CODEOWNERS file is valid.